File: Front/InvestQ-App/src/app/models/ClienteCorretora.ts
import { Cliente } from "./Cliente";
import { Corretora } from "./Corretora";
import { Guid } from 'guid-typescript';

export interface ClienteCorretora {

  clienteId: Guid;
  cliente: Cliente;
  corretoraId: Guid;
  corretora: Corretora;
}

File: Back/src/InvestQ.Domain/Entities/Clientes/ClienteCorretora.cs
using System;

namespace InvestQ.Domain.Entities.Clientes
{
    public class ClienteCorretora
    {
        public ClienteCorretora() { }
        public ClienteCorretora(Guid clienteId, 
                                Guid corretoraId) 
        {
            this.ClienteId = clienteId;
            this.CorretoraId = corretoraId;   
        }
        public Guid ClienteId { get; set; }
        public Cliente Cliente { get; set; }
        public Guid CorretoraId { get; set; }
        public Corretora Corretora { get; set; }
    }
}